app/build.gradle.kts

```diff
@@ -146,7 +146,7 @@ android {
                 "**.txt",
                 "**.bin",
                 "**.{html,mmd}", // Compose Destination
-                "**.properties", // Apache 5 HC version info
+                "okhttp3", // Okhttp public suffix
```

Comment:
Does not work, use /**.

Reply:
I doube if we can remove it?
Seems okhttp internal use it to parse SET-Cookie header?

Reply:
Ain't cookies handled in the ktor layer?

Reply:
> Ain't cookies handled in the ktor layer?

Yeah, you are right

Reply:
Done